Tiled/Loader.cs

```diff
@@ -25,8 +25,8 @@ public static class Loader
         /// <param name="path">The path where the tilemap can be found</param>
         /// <returns>Structured <see cref="Template"/></returns>
         public static Template LoadTemplate(string path) => LoadGeneric<Template>(path);
-        
-        public static T LoadGeneric<T>(string path)
+
+        private static T LoadGeneric<T>(string path)
```

Comment:
why did yo change it from public to private

Reply:
It's not something that's used by the end-user, and I'd rather we create functions to handle loading the specific data rather than exposing internal functionality to the end-user. 

Hel.Tiled is meant to be a black-box usage kind of library, you give us a JSON file and we give you a populated object with helper methods to use that object. This seems like a very white-box method, as we're exposing things unrelated to `Tiled` which may potentially confuse users, or lead to unexpected errors if they misuse it. 

Essentially, I changed it to `private` to abstract away internal functionality that may lead to more harm than good, as there's little reason for an end-user to choose `LoadGeneric<>` over `LoadTilemap`